src/renderers/shared/fiber/__tests__/ReactIncrementalSideEffects-test.js

```diff
@@ -1069,7 +1069,7 @@ describe('ReactIncrementalSideEffects', () => {
   });
 
   it('invokes ref callbacks after insertion/update/unmount', () => {
-
+    spyOn(console, 'error');
```

Comment:
If the warning wasn't intentional in this test we should fix the reason it occurs. If it was intentional we should assert on the `console.error` message.